blog/electron-32-0.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Electron 32.0.0
+date: 2024-08-20T00:00:00.000Z
+authors: VerteDinde
+slug: electron-32-0
+tags: [release]
+---
+
+Electron 32.0.0 has been released! It includes upgrades to Chromium `128.0.6613.36`, V8 `12.8`, and Node `20.16.0`.
+
+---
+
+The Electron team is excited to announce the release of Electron 32.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- Added new API version history in our documentation, a feature created by @piotrpdev as part of Google Summer of Code. [#42982](https://github.com/electron/electron/pull/42982)
```

Comment:
Note for reviewers: we have a draft blog post by @piotrpdev describing the new API history feature: https://github.com/electron/website/pull/627/files Depending on merge order, we should link that post here as well 😄

Reply:
Quick note: We're going to amend this post after publishing this - this post will publish first, then Piotr's blog 👍